### ISSUE TITLE
Support Ember Changeset v4

### ISSUE THREAD

**Issue by jelhan** [ember-bootstrap/ember-bootstrap-changeset-validations#40]
Ember Changeset v4, which has been released recently, should be supported. It is currently locked to v3: https://github.com/kaliber5/ember-bootstrap-changeset-validations/blob/d11f9cf84c467f9bcfa001763fb13613f50244d1/package.json#L31

We might be able to widen the SemVer range to allow v3 and v4 at the same time. That would be the best solution in my perspective. When doing so we should ensure that we run our tests for both versions.

If that is not possible, we should drop support for v3 and release a new major.

**Comment by simonihmig** [comment]
@jelhan we have a major update scheduled for this addon. Which is needed to support the upcomming release of ember-bootstrap 5.1. So do we want to cut another major a bit later, for this issue? Or do you have time to work on this in the near future? (I won't)

**Comment by simonihmig** [comment]
Thinking about this again, I guess it is not a good idea to require users to update to `ember-changeset-validations` v4 at the same time as requiring `ember-bootstrap` v5, *and* fixing the BsForm class issue for 5.1, as that would impose greater upgrade pain. So I'll release what we have now, and when we choose to drop `ember-changeset-validations` v3 in favor of v4, we can just do another major bump. It's not that we run out of version numbers... :wink:

**Comment by basz** [comment]
I now force v4 via a package resolution option, but it would be nice if I could drop that.

**Comment by basz** [comment]
`ember try:each`

```
Error!
Error: Cannot copy '../../../errno/cli.js' to a subdirectory of itself, '../../../errno/cli.js'.
Error: Cannot copy '../../../errno/cli.js' to a subdirectory of itself, '../../../errno/cli.js'.
    at /Users/bas/Sandbox/ember-bootstrap-changeset-validations/node_modules/ember-try/node_modules/fs-extra/lib/copy/copy.js:210:21
    at FSReqCallback.oncomplete (node:fs:188:23)
```
Which seems fixed via one of; https://github.com/jprichardson/node-fs-extra/issues?q=is%3Aissue+to+a+subdirectory+of+itself+is%3Aclosed

however setting a resolution to "10.1.0" (latest) does not fix it... (on MacOS).

**Comment by jelhan** [comment]
Support for Ember Changeset v4 has landed in [v5.0.0](https://github.com/ember-bootstrap/ember-bootstrap-changeset-validations/blob/master/CHANGELOG.md#v500-2022-04-29).
